src/data/polynomial/derivative.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Chris Hughes, Johannes Hölzl, Scott Morrison, Jens Wagemaker
 -/
 import data.polynomial.eval
+import tactic.ring
```

Comment:
You probably don't need this import anymore.

Reply:
I always forget this, thank you.